modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -399,5 +401,18 @@ def remove_special_issues_from_secondary_disabilities
         end
       end
     end
+
+    def change_of_address_provided?
+      form_data['veteran']['changeOfAddress'].present?
+    end
```

Comment:
not worth removing in this PR, but abstracting all these `.present?` checks into helper methods is starting to feel like unnecessary bloat to me

Reply:
Fair enough.
I think the original intent was that I just hated the way it would read if it just said,
`form_data['veteran']['changeOfAddress'] = transform_change_of_address_type_case`
because that feels like it would always be changing something. When in reality, it's only going to change it if it exists.  So then I start adding the little check that would try to signal that, "_we're gonna change this thing, only if the condition is right_". 🤷